### PR TITLE
Release 0.25.0

### PR DESCRIPTION
**Pull request by minevala**
## Description
New release 0.25.0

### CODE REVIEW

File: site/docs/about/roadmap.mdx [hunk 1]
```diff
@@ -25,12 +25,9 @@ For previously released features, see changelogs in the [What is new section](/a
 ~~- Component: Time input field~~   
 ~~- Component: Number input field~~   
 ~~- Icons: Next set of icons~~   
-
-
-
-- Component: Dialog
+~~- Component: Dialog~~
```
Comment (laurakarhu) [inline site/docs/about/roadmap.mdx:28]: Seems that the line change here will break in the actual view. Three white spaces in the end of the line does the trick in markdown. ( I tremble with the same problem every time)

**Review by laurakarhu — APPROVED** [review]
One tiny end of line change and it's perfect :)

File: site/docs/about/roadmap.mdx (hunk 1 above)
Comment (ronijaakkola) [reply site/docs/about/roadmap.mdx:28]: I can fix this.

Comment (ronijaakkola) [reply site/docs/about/roadmap.mdx]: ✅

**Review by ronijaakkola — APPROVED** [review]

File: site/docs/about/roadmap.mdx (hunk 1 above)
Comment (minevala) [reply site/docs/about/roadmap.mdx]: My editor did not like the three spaces 😬  I replaced them with good ol' <br/> - tag 😁

**Review by VilleMiekkoja — APPROVED** [review]
Looks good 🎉

**Pull request merged**